app/src/main/java/com/blueprint/featureA/data/model/local/FeatureAEntity.kt

```diff
@@ -5,6 +5,6 @@ import androidx.room.PrimaryKey
 
 @Entity(tableName = "feature_a")
 data class FeatureAEntity(
-    @PrimaryKey(autoGenerate = true) val id: Int,
+    @PrimaryKey(autoGenerate = true) val id: Int = 0,
```

Comment:
Jel nam treba ovaj fallback ako je `autogenerate`? 🤔

Reply:
Ako izostavis default vrednost, onda trazi da prosledis id kao param :/
mada nisam testirao radi li to uopste 🤔 (sad cu)

Reply:
update: radi autogenerate